ydb/docs/en/core/reference/observability/metrics/index.md

```diff
@@ -106,3 +618,350 @@ You can analyze a transaction's execution time using a histogram counter. The in
+
+Labels:
+
+- _topic_ – the name of the topic.
+- _consumer_ – the name of the consumer.
+
+||
+
+||
+
+`topic.partition.read.lag_milliseconds_max`
+`GAUGE`, milliseconds
+|
+The difference between the current time and the minimum write time among all of the messages read for the last minute in all partitions.
+
+Labels:
+
+- _topic_ – the name of the topic.
+- _consumer_ – the name of the consumer.
+
+||
+
+||
+
+`topic.partition.read.idle_milliseconds_max`
+`GAUGE`, milliseconds
+|
+The maximum idle time (the time since the last read operation from a partition) in all partitions.
+
+Labels:
+
+- _topic_ – the name of the topic.
+- _consumer_ – the name of the consumer.
+
+||
+
+||
+
+`topic.partition.read.lag_milliseconds_max`
+`GAUGE`, milliseconds
+|
+The maximum difference between the write time and the creation time among all of the messages that were read for the last minute.
+
+Labels:
+
+- _topic_ – the name of the topic.
+- _consumer_ – the name of the consumer.
+
+||
+
+||
+
+`topic.partition.write.lag_milliseconds_max`
+`GAUGE`, milliseconds
+|
+The maximum difference between the write time and the creation time among all of the messages that were written for the last minute.
+
+Labels:
+
+- _topic_ – the name of the topic.
+
+||
+
+||
+`topic.partition.write.speed_limit_bytes_per_second`
+`GAUGE`, bytes per second
+|
+The limit for the writing speed to a single partition.
+
+Labels:
+
+- _topic_ – the name of the topic.
+
+||
+
+||
+
+`topic.partition.write.throttled_nanoseconds_max`
+`GAUGE`, nanoseconds
+|
+The maximum throttled time for writing operations by all partitions. The worst case scenario, `topic.partition.write.throttled_nanoseconds_maх` = 10^9, means that writing was throttled for the whole second.
+
+Labels:
+
+- _topic_ – the name of the topic.
+
+||
+
+||
+
+`topic.partition.write.bytes_per_day_max`
+`GAUGE`, bytes
+|
+The maximum number of bytes written for the last day in all topic partitions.
+
+Labels:
+
+- _topic_ – the name of the topic.
+
+||
+
+||
+
+`topic.partition.write.bytes_per_hour_max`
+`GAUGE`, bytes
+|
+The maximum number of bytes written for the last hour in all topic partitions.
+
+Labels:
+
+- _topic_ – the name of the topic.
+
+||
+
+||
+
+`topic.partition.write.bytes_per_minute_max`
+`GAUGE`, bytes
+|
+The maximum number of bytes written for the last minute in all topic partitions.
+
+Labels:
+
+- _topic_ – the name of the topic.
+
+||
+
+||
+
+`topic.partition.write.idle_milliseconds_max`
+`GAUGE`, milliseconds
+|
+The maximum idle time of a partition for writing.
+
+Labels:
+
+- _topic_ – the name of the topic.
+
+||
+
+|#
+
+
+## Resource pool metrics {#resource_pools}
+
+#|
+||
+Metric name
+Type, units of measurement
+|
+Description
+||
+
+||
+`kqp.workload_manager.CpuQuotaManager.AverageLoadPercentage`
+`RATE`, pieces
+|
+Average database load. `DATABASE_LOAD_CPU_THRESHOLD` uses this metric.
```

Comment:
DATABASE_LOAD_CPU_THRESHOLD would better be a link to an explanation of what it is.